### ISSUE TITLE
stack(): argument 'tensors' (position 1) must be tuple of Tensors, not Proxy

### ISSUE THREAD

**Issue by leeqiaogithub** [pytorch/pytorch#74367]
### 🐛 Describe the bug

When I was doing pyTorch quantization, i met this issue:
tensor:stack(): argument 'tensors' (position 1) must be tuple of Tensors, not Proxy

code is as follows:
    coords = torch.meshgrid(range_ht, range_wd)
    coords = torch.stack(coords[::-1], dim=0).float()

I wonder why these basic operators are not support,such as torch.stack()  torch.tensor()  torch.arrange(). And how can i fix this issue

### Versions

Collecting environment information...
PyTorch version: 1.8.1
Is debug build: False
CUDA used to build PyTorch: 10.2
ROCM used to build PyTorch: N/A

OS: Ubuntu 16.04.6 LTS (x86_64)
GCC version: (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
Clang version: Could not collect
CMake version: version 3.5.1
Libc version: glibc-2.9

Python version: 3.7.0 (default, Oct  9 2018, 10:31:47)  [GCC 7.3.0] (64-bit runtime)
Python platform: Linux-4.4.0-210-generic-x86_64-with-debian-stretch-sid
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration: 
GPU 0: GeForce RTX 2080 Ti
GPU 1: GeForce RTX 2080 Ti
GPU 2: GeForce RTX 2080 Ti
GPU 3: GeForce RTX 2080 Ti

Nvidia driver version: 430.26
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.3
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.21.2
[pip3] pytorch-debayer==1.0.0
[pip3] pytorch-msssim==0.2.1
[pip3] torch==1.8.1
[pip3] torchaudio==0.8.0a0+e4e171a
[pip3] torchvision==0.9.1
[conda] blas                      1.0                         mkl    defaults
[conda] cudatoolkit               10.2.89              hfd86e86_1    defaults
[conda] ffmpeg                    4.3                  hf484d3e_0    pytorch
[conda] mkl                       2021.4.0           h06a4308_640    defaults
[conda] mkl-service               2.4.0            py37h7f8727e_0    defaults
[conda] mkl_fft                   1.3.1            py37hd3c417c_0    defaults
[conda] mkl_random                1.2.2            py37h51133e4_0    defaults
[conda] numpy                     1.21.2           py37h20f2e39_0    defaults
[conda] numpy-base                1.21.2           py37h79a1101_0    defaults
[conda] pytorch                   1.8.1           py3.7_cuda10.2_cudnn7.6.5_0    pytorch
[conda] pytorch-debayer           1.0.0                    pypi_0    pypi
[conda] pytorch-msssim            0.1                      pypi_0    pypi
[conda] torchaudio                0.8.1                      py37    pytorch
[conda] torchvision               0.9.1                py37_cu102    pytorch

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo

**Comment by jingyu-ml** [comment]
do you solve this?

**Comment by leeqiaogithub** [comment]
> do you solve this?

No, I just avoided it.

**Comment by jerryzh168** [comment]
there seems to be some workaround in fx, but I'm not exactly sure what it is, we are moving to new flow though so this is probably not going to be fixed

**Comment by jerryzh168** [comment]
closing for now, let me know if you have any further questions
